Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -92,11 +87,29 @@ class TextureCacheBase
       other_entry->references.emplace(this);
     }
 
+    enum FlagPos
+    {
+      EFB_COPY_FLAG_POS = 0,
```

Comment:
Eh..someone else on IRC mentioned that too. I'm not a huge fan of bitfield.  I can't remember where it was but something I had to do on Vertex Rounding really turned me off to them.  They feel archaic and can be painful to update when dealing with many attributes (not to mention dealing with padding and such).  I'll look at them some more though and see if I can get on board with them.

I can't speak to bitset much either to be fair, I've only used it a few times and this was my first time using it for flags.  I think it's more eloquent than bitfields but figure they aren't always interchangeable.

Honestly, I'm leaning more toward going back to bools.  Did you do some profiling in your initial PR?  Why did you decide to move to a single value for format + flags?